doc/db_schema.md

```diff
@@ -22,11 +22,14 @@ erDiagram
 
     Certficate ||--o{ Slice : has
 
-    WalletSection {
-        uuid Id PK "Unique id of a section"
-        uuid WalletId FK "The wallet that owns the section"
-        int WalletPosition "The position of the section in the wallet"
-        bytea PublicKey "The public key of the section, generated from the privatekey and Wallet position"
+    DepositEndpoint {
+        uuid Id PK "Unique id of a deposit endpoint"
+        uuid WalletId FK "The wallet that owns the deposit endpoint, if null then it is a ReceiverDepositEndpoint"
+        int WalletPosition "The position of the deposit endpoint in the wallet"
+        bytea PublicKey "The public key of the deposit endpoint, generated from the privatekey and Wallet position"
+        text Owner "Identifies the owner, is the same as subject from the JWT"
+        text ReferenceText "Textural reference of the depositEndpoint"
+        text Endpoint "The URL of where the wallet system of receiver deposit endpoint is placed"
     }
 
     Slice {
```

Comment:
Same with the blinding factor, is it safe to store it?

Reply:
It is required to be stored as of now.

We have had discussions in the ProjectOrigin forum wether or not this could possible be generated deterministically, but it would **not** make it more secure. The data required to generate it would still be stored in the database, and the algorithm would be publicly available since it is OpenSource, so any user with access would still be able to generate it.